data.midrc.org/portal/gitops.json

```
@@ -249,13 +249,13 @@
             ],
             "guppyConfig": {
               "dataType": "study",
-              "nodeCountTitle": "Studies",
+              "nodeCountTitle": "Datasets",
               "fieldMapping": [],
               "manifestMapping": {
                 "resourceIndexType": "imaging_data_file",
                 "resourceIdField": "object_id",
-                "referenceIdFieldInResourceIndex": "id",
-                "referenceIdFieldInDataIndex": "study_id"
+                "referenceIdFieldInResourceIndex": "study_id",
```

```suggestion
                "referenceIdFieldInResourceIndex": "_study_id",
```
you have changed this in etlmapping as well